### PR TITLE
Add plugin keybinding and api test cases

### PR DESCRIPTION
**Pull request by peterfication**
With these test cases we should be able to prevent breaking changes :)

### CODE REVIEW

**Review by johmsalas — APPROVED** [review]
I truly appreciate all your contributions; they are marking a before and after for this plugin and have given me the motivation that I had lost to work on it.

**Comment by johmsalas** [comment]
@peterfication To avoid working on the same stuff, I created a couple issues. If you are doing for example Integration Tests for LSP https://github.com/johmsalas/text-case.nvim/issues/61 or Telescope https://github.com/johmsalas/text-case.nvim/issues/62, you can assign the issues to yourself

If you have not started those, this is something I'd like to actually do myself :), that's why I already assign them to me. But won't start until you confirm that's not WIP

File: README.md [hunk 1]
```diff
@@ -158,3 +158,6 @@ A requirement for LSP rename to work is to have LSP set in the buffer and the La
 
 To triage it, trigger LSP renaming using `:lua vim.lsp.buf.rename()` while the cursor is on the symbol. If it works, file an issue on this plugin
 
+## Development
```
[the remark this answers is not in the document]
Comment (peterfication) [reply README.md:161]: I haven't tried `Act` in a long time. I just run against my normal Neovim instance. But I see that this won't work if you want to test against Neovim nightly.

With asdf, you can temporarily overwrite a tool version e.g.

```
ASDF_NEOVIM_VERSION=nigthly ./tests/run.sh
```

And you need to change the `.tool-versions` line `neovim 0.9.4` to `neovim 0.9.4 nightly`.

However, it's not that straight forward for Neovim it seems, at least I can't get it to run on my machine properly.

[the remark this answers is not in the document]
Comment (peterfication) [reply README.md:161]: Act might ignore it, because the event is not `push` or `pull_request` but `cronjob`.

**Comment by peterfication** [comment]
> I truly appreciate all your contributions; they are marking a before and after for this plugin and have given me the motivation that I had lost to work on it.

That's really cool to hear! I enjoy diving into the world of Lua and Neovim plugins, so I understand the editor better I'm working with everyday :) And some other learnings along the way :)

**Comment by peterfication** [comment]
> @peterfication To avoid working on the same stuff, I created a couple issues. If you are doing for example Integration Tests for LSP #61 or Telescope #62, you can assign the issues to yourself
> 
> If you have not started those, this is something I'd like to actually do myself :), that's why I already assign them to me. But won't start until you confirm that's not WIP

Yes, it's a good idea to work with issues from now on to align better :)

I haven't started working on LSP or Telescope tests. They probably require adjustments in the `tests/init.lua` so these plugins are loaded properly. Go ahead with them. I added another test issue: https://github.com/johmsalas/text-case.nvim/issues/63 for tests for visual ranges and I will work on that then.

**Pull request merged**